crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -49,6 +50,39 @@ pub struct CheckpointResponse {
 
 // The constituent parts of checkpoints, signed and certified
 
+/// The Sha512 digest of an EllipticCurveMultisetHash committing to the live object set.
```

Comment:
Then we should change that comment  to sha256, wondering if we should actually use a faster primitive, ie blake2

Reply:
Oh sorry, I copied this from somewhere else, obviously its wrong.

As far as I know this hash is only used to hash the accumulator at the end, it isn't used for hashing inputs to points, so it shouldn't matter.